src/SmartComponents/RunTaskModal/ConversionTaskInputParameters.js

```diff
@@ -0,0 +1,197 @@
+    </a>
+  );
+
+  const inPlaceSupportLink = (
+    <a
+      href="https://access.redhat.com/support/policy/ipu-support"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      In-place upgrade Support Policy
+    </a>
+  );
+
+  const analysisTitleSection = (
+    <TextContent>
+      <Text component={TextVariants.h4}>
+        Analyze conversion to RHEL 7 without Extended Lifecycle Support (ELS)
+      </Text>
+      <Text component={TextVariants.p}>
+        By default, the task analyzes a conversion of your CentOS Linux 7 to a
+        supported RHEL 7 system with the latest security patches and updates via
+        ELS. {readMoreLink}
+      </Text>
+    </TextContent>
+  );
+
+  const convertTitleSection = (
+    <TextContent>
+      <Text component={TextVariants.h4}>
+        Convert to RHEL 7 without Extended Lifecycle Support (ELS)
+      </Text>
+      <Text component={TextVariants.p}>
+        By default, the task converts your CentOS Linux 7 to a supported RHEL 7
+        system with the latest security patches and updates via ELS.{' '}
+        {readMoreLink}
+      </Text>
+    </TextContent>
+  );
+
+  return (
+    <Form className="pf-v5-u-pb-lg">
+      {slug.includes('analysis') ? analysisTitleSection : convertTitleSection}
+
+      <Checkbox
+        id={elsDisabled.key}
+        label={elsDisabled.title}
+        isChecked={toBool(elsDisabledValue)}
+        onChange={(_event, value) => {
+          const newValue = value ? 'True' : 'False';
+          setElsDisabledValue(newValue);
+          updateParameters(elsDisabled, newValue);
+        }}
+      />
+      <TextContent>
+        <Text
+          component={TextVariants.p}
+          style={{
+            fontSize: '14px',
+            color: '#6a6e73',
+            paddingLeft: '20px',
+            marginTop: '-20px',
+          }}
+        >
+          If you plan to upgrade to RHEL 8 right after the conversion, you may
+          opt not to use the ELS subscription. Note that the conversion and the
+          subsequent upgrade without an ELS subscription come with a limited
+          support scope per the {convert2RHELSupportLink} and the{' '}
+          {inPlaceSupportLink}.
+        </Text>
+      </TextContent>
+
+      <TextContent>
+        <Text component={TextVariants.h4}>
+          Ignore specific pre-conversion analysis check results.
+        </Text>
+        <div style={{ border: 'thin solid black', padding: '5px 10px' }}>
+          <span style={{ fontWeight: 'bold' }}>
+            <Icon status={'warning'}>
+              <ExclamationTriangleIcon />
+            </Icon>{' '}
+            Resolve issues if possible.
+          </span>
+          <Text component={TextVariants.p}>
+            Use these at your discretion and resolve the issues if possible.
+            Overriding issues can cause the conversion to fail, resulting in a
+            deteriorated system state.
+          </Text>
+        </div>
```

Comment:
```suggestion
        <Alert variant="warning" isInline title="Resolve issues if possible">
          <Text component={TextVariants.p}>
            Use these at your discretion and resolve the issues if possible.
            Overriding issues can cause the conversion to fail, resulting in a
            deteriorated system state.
          </Text>
        </Alert>
```

Reply:
Ah yes, an Alert component is much more appropriate and nicer looking than my boxed monstrosity.  Done!